homeassistant/components/advantage_air/sensor.py

```diff
@@ -71,21 +75,21 @@ def __init__(self, instance, ac_key, action):
         self._attr_unique_id += f"-timeto{action}"
 
     @property
-    def native_value(self):
+    def native_value(self) -> Decimal:
         """Return the current value."""
         return self._ac[self._time_key]
 
     @property
-    def icon(self):
+    def icon(self) -> str:
         """Return a representative icon of the timer."""
         if self._ac[self._time_key] > 0:
             return "mdi:timer-outline"
         return "mdi:timer-off-outline"
 
-    async def set_time_to(self, **kwargs):
+    async def set_time_to(self, **kwargs) -> None:
```

Comment:
I think this is the last one.
```suggestion
    async def set_time_to(self, **kwargs: Any) -> None:
```